index.js

```diff
@@ -296,11 +297,17 @@ const GetEmail = {
         console.log('get email intent fired');
         if (send) {
             updateAWSConfig();
-            // This promise might be broken.
-            const sendPromise = queueSendEmail();
+            const sendPromise = new AWS.SES({apiVersion: '2010-12-01'}).sendEmail(emailClient.generateParams(userName, data)).promise();
+            sendPromise.then(
+              function(data) {
+                console.log(data.MessageId);
```

Comment:
Should we `resetToInitialState()` within the `.then()` statement? Now that the skill is completed and the data is sent, we need the next report to be clear.

Reply:
yes that would be safe